snforge_std/src/cheatcodes/tx_info.cairo

```diff
@@ -70,10 +65,5 @@ fn stop_spoof(target: CheatTarget) {
 }
 
 fn extend_array(ref array: Array<felt252>, mut span: Span<felt252>) {
-    loop {
-        match span.pop_front() {
-            Option::Some(x) => { array.append(x.clone()); },
-            Option::None => { break; }
-        };
-    };
+    array.append_span(span);
```

Comment:
Can we remove `extend_array` altogether and just use this method instead?

Reply:
sure!